corehq/middleware.py

```diff
@@ -146,7 +146,7 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             timeout = settings.SECURE_TIMEOUT
             # force re-authentication if the user has been logged in longer than the secure timeout
             if self._session_expired(timeout, request.user.last_login, now):
-                django_logout(request, template_name=settings.BASE_TEMPLATE)
+                LogoutView.as_view(template_name=settings.BASE_TEMPLATE)(request)
```

Comment:
I was unable to find clear instructions for how to replace `django_logout`, so I'm not certain that this is exactly equivalent.

Reply:
Seems to be right:
```
The LoginView and LogoutView class-based views supersede the deprecated login() and logout() function-based views.
```

Also looking at the code for `logout` it's doing the same thing as you are here.